presto-cli/src/test/java/com/facebook/presto/cli/TestClientOptions.java

```diff
@@ -143,27 +143,27 @@ public void testUpdateSessionParameters()
         ClientSession session = options.toClientSession();
         SqlParser sqlParser = new SqlParser();
 
-        ImmutableMap<String, String> existingProperties = ImmutableMap.of("query_max_memory", "10GB", "distributed_join", "true");
```

Comment:
Spoke to @KBP-TDC and Martin.  They said we should get rid of the old property altogether.  I'm going to change it back.